AntennaTracker/GCS_Mavlink.cpp

```diff
@@ -369,6 +369,13 @@ MAV_RESULT GCS_MAVLINK_Tracker::handle_command_long_packet(const mavlink_command
     }
 }
 
+bool GCS_MAVLINK_Tracker::set_home_to_current_location(bool lock) {
+    return tracker.set_home(AP::gps().location());
```

Comment:
Possibly a change in behavior but we actually moved away from using GPS location for this on plane, as it resulted in a weird looking updates when you update home to the current location but it doesn't actually match the reported position. The fix is to just ask AHRS where we are instead. This form of GPS access is also bad because it's whatever the last GPS was, but we could have lost fix and be looking at a very old position here.

Reply:
No regression here as this is a new implementation.

No question we should do better, as discussed elsewhere.